tests/TestCase/Shell/CompletionShellTest.php

```diff
@@ -116,7 +116,7 @@ public function testCommands()
         $output = $this->out->output;
 
         $expected = 'TestPlugin.example TestPlugin.sample TestPluginTwo.example unique welcome ' .
-            "cache i18n orm_cache plugin routes server i18m sample testing_dispatch\n";
+            "cache i18n orm_cache plugin routes server version i18m sample testing_dispatch\n";
```

Comment:
I think I'm the one who added this, but it was a while ago (maybe years ?).

If I remember correctly it was for a test case linked to the `CompletionShell`. However, the details elude me now...